src/c++/perf_analyzer/docs/README.md

```diff
@@ -29,7 +29,7 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 # **Perf Analyzer Documentation**
 
 | [Installation](README.md#installation) | [Getting Started](README.md#getting-started) | [User Guide](README.md#user-guide) |
-| -------------------------------------- | -------------------------------------- | -------------------------------------- |
+| -------------------------------------- | -------------------------------------------- | ---------------------------------- |
```

Comment:
Why was this changed?

Reply:
actually it looks like my vscode formatter lined up the changed line with the line above it.